AutoCollection/diagnostic-channel/mysql.sub.ts

```diff
@@ -17,11 +17,16 @@ export const subscriber = (event: IStandardEvent<mysql.IMysqlData>) => {
         const connectionConfig = connection.config || {};
         const dbName = connectionConfig.socketPath ? connectionConfig.socketPath : `${connectionConfig.host || "localhost"}:${connectionConfig.port}`;
         client.trackDependency(
-                dbName,
-                sqlString,
-                event.data.duration | 0,
-                success,
-                "mysql");
+            {
+                target: dbName,
+                data: sqlString,
+                name: sqlString,
+                duration: event.data.duration | 0,

```

Comment:
Is this guard necessary? `trackDependency` should defend against NaN duration

Reply:
Removed the guard. Thinking about it, it is not just unnecessary, but incorrect, because duration could be missing  in case of incomplete dependency that is being logged and we shouldn't be treating every empty value as 0. We are doing it now (for all SDKs) but should address it separately.